### DISCUSSION TITLE
VSCode extension fails with: Extension activation failed: "ENOENT: no such file or directory, uv_cwd"

### DISCUSSION THREAD

**Discussion by apoloval** [community/community#74001] [Copilot Conversations]
### Select Topic Area

Bug

### Body

The extension has been working like a charm for weeks. But suddenly, the plugin fails at launch with the following message:

`
[ERROR] [default] [2023-11-07T09:40:28.935Z] Extension activation failed: "ENOENT: no such file or directory, uv_cwd"
`

Plugin version is v1.134.0, VSCode is 1.84.0. I cannot find any information about this error anywhere, including this forum. Only references in Google to other projects that fails because of the same error produced in their JS code. 

Please any help?

**Comment by apoloval** [comment]
Wow! After a system restart, the error has disappeared! That was the only thing I didn't try.

**Comment by devaniljr** [comment]
I'm getting a lot of this error. When I restart it works. But it's frustrating to do it all the time. There is a cause and a possible solution for this?

**Comment by paoloose** [comment]
Running `killall code` on Linux solved it for me.

**Reply by shinebayar-g** [reply]
This is good to know! Thanks.

**Reply by wandrey7** [reply]
TKS, it's working for me
